Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -650,9 +650,14 @@ - (instancetype)initWithLogFileManager:(id <DDLogFileManager>)aLogFileManager
 - (void)lt_cleanup {
     NSAssert([self isOnInternalLoggerQueue], @"lt_ methods should be on logger queue.");
 
-    [_currentLogFileHandle synchronizeFile];
-    [_currentLogFileHandle closeFile];
-
+    if (@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)) {
+        [_currentLogFileHandle synchronizeAndReturnError:nil];
```

Comment:
Are you sure that we need to hide possible error?
I would like to know if something strange has happened.

Reply:
Done!